phpstan-baseline.neon

```diff
@@ -9,3 +9,28 @@ parameters:
             message: "#^Call to static method getInstance\\(\\) on an unknown class Sonata\\\\EasyExtendsBundle\\\\Mapper\\\\DoctrineCollector\\.$#"
             count: 1
             path: src/DependencyInjection/SonataMediaExtension.php
+
+        -
+            message: "#^Call to an undefined method Sonata\\\\ClassificationBundle\\\\Model\\\\CategoryManagerInterface\\:\\:getRootCategories\\(\\)\\.$#"
+            count: 1
+            path: src/Model/CategoryManager.php
+
+        -
+            message: "#^Call to an undefined method Sonata\\\\ClassificationBundle\\\\Model\\\\CategoryManagerInterface\\:\\:getRootCategory\\(\\)\\.$#"
+            count: 1
+            path: src/Model/CategoryManager.php
+
+        -
+            message: "#^Call to an undefined method Twig\\\\Environment\\:\\:expects\\(\\)\\.$#"
+            count: 1
+            path: tests/Block/GalleryBlockServiceTest.php
```

Comment:
We should fix this error here: 
https://github.com/sonata-project/SonataBlockBundle/blob/3.x/src/Test/BlockServiceTestCase.php#L50-L53